partners/astropy.md

```diff
@@ -0,0 +1,42 @@
+# pyOpenSci's partnership with Astropy
+
+:::{figure-md} astropy-logo
+:class: myclass
+
+<img src="/images/astropy_project_logo.svg" alt="The astropy logo. The logo is an orange egg shape with a white snake spiraling in it. The text of the image says The Astropy Project on three lines." class="mb-1" width="400px">
+
+...
+:::
+
+pyOpenSci has a collaboration with the [Astropy scientific community](https://www.astropy.org/). Through this collaboration we add an additional layer of peer review checks to Astropy-related packages that go through our [open peer review process](https://www.pyopensci.org/software-peer-review/about/intro.html). If the package meets both the Astropy criteria and our pyOpenSci peer review criteria, it can then become both an astropy affiliated package and a pyOpenSci ecosystem package.
+
+If you are submitting a package for review and wish to also apply for astropy affiliation, your package will be:
+Reviewed against current pyOpenSci guidelines and checks.
+Also reviewed against the Astropy-specific guidelines listed below.
+
+If you are submitting a package for review and wish to also apply for astropy affiliation, your package will be:
+
+- Reviewed against current pyOpenSci guidelines and checks.
+- Also reviewed against the Astropy-specific guidelines listed below.
+
+(astropy-collaboration)=
```

Comment:
I didn't know you could do this in Markdown.  Cool!

Reply:
it might be a [superpower of MyST ](https://mystmd.org/). i really love MyST for all of our documentation and anything sphinx related. it's such a game changer in terms of flexibility / readability with simple syntax!

Reply:
oh @pllim that is a [myst target](https://jupyterbook.org/en/stable/reference/cheatsheet.html#target-headers). it's a nice way to have a consistent target link in case someone changes header text and forgets and also to just make a nice short target link. so you can think of it as a shortcut to make cross linking to section headers within a sphinx document less error prone. i'm not sure it's visible in the final rendered page however just internally it's helpful in case people change header names.